### PR TITLE
Update Protocol.Params.Param.Interprete.md

### PR DESCRIPTION
**Pull request by RobbieDemaegdt**

### CODE REVIEW

**Comment by EdithLansens** [comment]
@PedroDebevere, could you review this pull request?

File: develop/schemadoc/Protocol/Protocol.Params.Param.Interprete.md [hunk 1]
```diff
@@ -6,6 +6,36 @@ uid: Protocol.Params.Param.Interprete
 
 Specifies how a parameter value is processed.
 
+By default, try to follow the following guidelines for Interpretes and Measurements:
```
Comment (PedroDebevere) [inline develop/schemadoc/Protocol/Protocol.Params.Param.Interprete.md:9]:
```suggestion
By default, try to follow the following guidelines for specifying values for the Interprete and Measurement subtags:
```

File: develop/schemadoc/Protocol/Protocol.Params.Param.Interprete.md [hunk 2]
```diff
@@ -6,6 +6,36 @@ uid: Protocol.Params.Param.Interprete
...
+
+  - Interprete.Type = double
+
+  - Measurement.Type = number 
+
+- **For tables**:
+
+  - Interprete.Rawtype = other
+
+  - Interprete.Type = double
```
Comment (PedroDebevere) [inline develop/schemadoc/Protocol/Protocol.Params.Param.Interprete.md:31]:
```suggestion
```
remove this line (for tables, the DIS snippets do not include an Interprete tag)

File: develop/schemadoc/Protocol/Protocol.Params.Param.Interprete.md [hunk 3]
```diff
@@ -6,6 +6,36 @@ uid: Protocol.Params.Param.Interprete
...
+
+  - Interprete.Rawtype = numeric text
+
+  - Interprete.Type = double
+
+  - Measurement.Type = number 
+
+- **For tables**:
+
+  - Interprete.Rawtype = other
```
Comment (PedroDebevere) [inline develop/schemadoc/Protocol/Protocol.Params.Param.Interprete.md:29]:
```suggestion
```
remove this line (for tables, the DIS snippets do not include an Interprete tag)

File: develop/schemadoc/Protocol/Protocol.Params.Param.Interprete.md [hunk 4]
```diff
@@ -6,6 +6,36 @@ uid: Protocol.Params.Param.Interprete
...
+
+  - Interprete.Type = double
+
+  - Measurement.Type = number
```
Comment (PedroDebevere) [inline develop/schemadoc/Protocol/Protocol.Params.Param.Interprete.md:33]:
```suggestion
  - Measurement.Type = table
```

File: develop/schemadoc/Protocol/Protocol.Params.Param.Interprete.md [hunk 5]
```diff
@@ -6,6 +6,36 @@ uid: Protocol.Params.Param.Interprete
...
+
+  - Interprete.Rawtype = other
+
+  - Interprete.Type = double
+
+  - Measurement.Type = number 
+
+- **For discreets**:
+
+  If all the values in a discreet are numbers, it is better to use interpretes for numbers.
```
Comment (PedroDebevere) [inline develop/schemadoc/Protocol/Protocol.Params.Param.Interprete.md:37]:
```suggestion
  If all the values in a discreet are numbers, it is better to use Interprete.Type and Interprete.RawType as specified in **For numbers**.
```

Comment (EdithLansens) [reply develop/schemadoc/Protocol/Protocol.Params.Param.Interprete.md:35]:
```suggestion
  If all the values in a discreet are numbers, it is better to use Interprete.Type and Interprete.RawType as specified above in "for numbers".
```

**Review by EdithLansens — APPROVED** [review]

**Pull request merged**